scripts/build/build.mjs

```diff
@@ -150,34 +150,47 @@ async function preparePackage() {
 }
 
 async function run(params) {
-  await execa("rm", ["-rf", "dist"]);
-  await execa("mkdir", ["-p", "dist"]);
-  if (!params.playground) {
-    await execa("mkdir", ["-p", "dist/esm"]);
+  const shouldUseCache = !params.file && !params["purge-cache"];
+  const shouldPreparePackage = !params.playground && !params.file;
+  let configs = bundleConfigs;
+  if (params.file) {
+    configs = configs.filter(({ output }) => output === params.file);
+  } else {
+    await execa("rm", ["-rf", "dist"]);
   }
 
+  await execa("mkdir", ["-p", "dist"]);
+  await execa("mkdir", ["-p", "dist/esm"]);
```

Comment:
we should not make `dist/esm` for playground?
```suggestion
  if (!params.playground) {
    await execa("mkdir", ["-p", "dist/esm"]);
  }
```

Reply:
I don't think it matters to make an empty dir.